dev_docs/tutorials/saved_objects.mdx

```diff
@@ -257,3 +257,41 @@ the error should be verbose and informative so that the corrupt document can be
 ### Testing Migrations
 
 Bugs in a migration function cause downtime for our users and therefore have a very high impact. Follow the <DocLink id="kibDevTutorialTestingPlugins" section="saved-objects-migrations" text="Saved Object migrations section in the plugin testing guide"/>.
+
+### How can I do a "Runtime" migration?
```

Comment:
I find the section title a bit strange. Wouldn't something like `How to opt-out of the global savedObjects APIs` or something similar be more explicit?

Reply:
Thanks for the suggestion! I was trying to come up with something 'catchy' and relevant.